maintainer/benchmarks/benchmarks.py

```diff
@@ -72,13 +77,23 @@ def get_timings(system, n_steps, n_iterations, verbose=True):
 
     '''
     if verbose:
-        print(f"Timing every {n_steps} steps")
+        print(
+            f"Timing every {n_steps_integration} steps of integration and {n_steps_MC} MC steps of reactions")
+
+    if(n_steps_MC > 0 and RE is None):
+        raise Exception(
+            'For MC benchmarking RE = espressomd.reaction_ensemble.ReactionEnsemble must be provided')
+
     timings = []
     for i in range(n_iterations):
         tick = time.time()
-        system.integrator.run(n_steps)
+        if(n_steps_MC > 0):  
+            RE.reaction(n_steps_MC)
+        system.integrator.run(n_steps_integration)
         tock = time.time()
-        t = (tock - tick) / n_steps
+        # FIXME the MC and MD steps are not really additive, we should count
+        # each of them separately
+        t = (tock - tick) / (n_steps_integration + n_steps_MC)
```

Comment:
For now I have added the two different timings as two lines in the CSV output file. I think that it would be desirable in the future to add some kind of flag as you suggest so one can distinguish the two lines.

Reply:
I've added a 'label' column to distinguish MC from MD rows (0ec5b89cc0e8736).